api/status.go

```diff
@@ -1,9 +1,14 @@
 package api
 
+import (
+	"fmt"
+	"net/http"
+	"strings"
+)
+
 type Status struct {
-	Name       string `json:"name"`
-	Version    string `json:"version"`
-	APIVersion int    `json:"api_version"`
+	Name    string `json:"name"`
+	Version string `json:"version"`
```

Comment:
Can we just make the status API not auth-protected?

Reply:
We sure could. Seemed like you didn't want to before, though. Just wanna do it anyway now?

Reply:
Yeah.  We just need to remove the Version - set an omitempty, and dont set it if we aren't authenticated.

Reply:
I'll put APIVersion back here. The auth hiding isn't in scope here because... v2 doesn't know what auth is yet

Reply:
Right.  Unauthenticated sessions should receive:

```json
{
  "name"        : "My Awesome SHIELD",
  "api_version" : 2
}
```

Authenticated sessions should receive:

```json
{
  "name"        : "My Awesome SHIELD",
  "api_version" : 2,
  "version"     : "8.0.3"
}
```

Reply:
You know... I'll put it in here. We'll leave status unauthed and I'll do manual auth inside so we can have the special handling.